stingray/powerspectrum.py

```diff
@@ -885,7 +883,7 @@ def initial_checks(self, *args, **kwargs):
         return AveragedCrossspectrum.initial_checks(self, *args, **kwargs)
 
 
-class DynamicalPowerspectrum(AveragedPowerspectrum):
+class DynamicalPowerspectrum(DynamicalCrossspectrum):
```

Comment:
Also for this class I would specify the dt as a possible parameter if an EventList is passed as in the DynamicalCrossspectrum